core/commands/storage/stats/stats.go

```diff
@@ -161,6 +165,71 @@ This command get node stats in the network from the local node data store.`,
 	Type: nodepb.StorageStat{},
 }
 
+// sub-commands: btfs storage stats list
+var storageStatsListCmd = &cmds.Command{
+	Helptext: cmds.HelpText{
+		Tagline: "List node stats.",
+		ShortDescription: `
+This command list node stats in the network from the local node data store.`,
+	},
+	Arguments: []cmds.Argument{
+		cmds.StringArg("from", true, false, "list host local stats range from"),
+		cmds.StringArg("to", true, false, "list host local stats range to"),
+	},
+	RunTimeout: 30 * time.Second,
+	Run: func(req *cmds.Request, res cmds.ResponseEmitter, env cmds.Environment) error {
+		cfg, err := cmdenv.GetConfig(env)
+		if err != nil {
+			return err
+		}
+
+		n, err := cmdenv.GetNode(env)
+		if err != nil {
+			return err
+		}
+
+		from, err := strconv.ParseInt(req.Arguments[0], 10, 64)
+		if err != nil {
+			return err
+		}
+		to, err := strconv.ParseInt(req.Arguments[1], 10, 64)
+		if err != nil {
+			return err
+		}
+		list, err := ListHostStatsFromDatastore(req.Context, n, n.Identity.String(), from, to)
+		if err != nil {
+			return err
+		}
+
+		for _, hs := range list {
+			// Refresh latest repo stats
+			stat, err := corerepo.RepoStat(req.Context, n)
+			if err != nil {
+				return err
+			}
+
+			cfgRoot, err := cmdenv.GetConfigRoot(env)
+			if err != nil {
+				return err
+			}
+			du, err := disk.Usage(cfgRoot)
+			if err != nil {
+				return err
+			}
+
+			hs.Stat.Online = cfg.Experimental.StorageHostEnabled
+			hs.Stat.StorageUsed = int64(stat.RepoSize)
+			hs.Stat.StorageCap = int64(stat.StorageMax)
+			hs.Stat.StorageDiskTotal = int64(du.Total)
+			hs.Stat.StorageDiskAvailable = int64(du.Free)
```

Comment:
Should not call this, it should give whatever has been stored in the datastore - otherwise all stats have the same *latest* stats